test/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter_test.cc

```diff
@@ -250,6 +250,26 @@ TEST_F(GrpcHttp1BridgeFilterTest, HandlingBadGrpcStatus) {
   EXPECT_EQ("foo", response_headers.get_("grpc-message"));
 }
 
+// Verifies that we convert grpc-status to the appropriates status code for a headers only response
+TEST_F(GrpcHttp1BridgeFilterTest, HandlingHeadersOnlyResponseBadGrpcStatus) {
```

Comment:
Can we have a test that verifies that the old behavior is preserved when the runtime flag is set? Thanks

Reply:
ok